modules/account-lib/src/coin/algo/utils.ts

```diff
@@ -556,12 +571,70 @@ export class Utils implements BaseUtils {
     if (!key || typeof key !== 'string') {
       throw new Error('Invalid base64 string');
     }
-    const base64RegExp =
-      /^(?:[a-zA-Z0-9+\/]{4})*(?:|(?:[a-zA-Z0-9+\/]{3}=)|(?:[a-zA-Z0-9+\/]{2}==)|(?:[a-zA-Z0-9+\/]{1}===))$/;
+
     if (!base64RegExp.test(key)) {
       throw new Error('Invalid base64 string');
     }
   }
+
+  /**
+   * Get Algorand asset with old or new token name.
+   * New token names have the tokens suffix in their names.
+   * The new format is 'algo:SUFFIX-ID'.
+   * The old format is 'algo:ID'.
+   * @param tokenNameOrId
+   */
+  getAsset(tokenNameOrId: string): Readonly<BaseCoin> {
+    return this._coins.has(tokenNameOrId)
+      ? this._coins.get(tokenNameOrId)
+      : this._coins.get(this.toOldTokenNameFormat(tokenNameOrId));
+  }
+
+  /**
+   * Convert a tokenName to the old format. If the name is already is in the old format,
+   * the same name is returned.
+   * @param tokenName a token name in the old or new format
+   */
+  toOldTokenNameFormat(tokenName: string): string {
+    return tokenName.replace(removeTokenSuffixRE, ':');
+  }
+
+  getAlgoAssets(): CoinMap {
+    if (!this._algoAssets) {
+      this._algoAssets = this._coins.filter((coin) => coin.network.family === CoinFamily.ALGO);
+    }
+
+    return this._algoAssets;
+  }
+
+  /**
+   * Convert a tokenName to the old format. If the name is already is in the old format,
+   * the same name is returned.
+   * @param tokenName a token name in the old or new format
```

Comment:
```suggestion
   * Convert a tokenName to the new format. If the name is already is in the new format,
   * the same name is returned.
   * @param tokenName a token name in the old or new format
```

Reply:
Add a comment to explain what if statics has old tokens. Because in that case you will not be able to get the new format.

Reply:
code was removed